testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -467,6 +469,21 @@ def test_none_filename(self, tmpdir):
             with hole2.HoleAnalysis(universe_none_filename) as h:
                 h.run()
 
+    def test_create_vmd_surface(self, universe, tmpdir):
+        errmsg = exe_err.format(name='dummy_path', kw='sph_process')
+        with pytest.raises(OSError, match=errmsg):
+            create_vmd_surface(sph_process='dummy_path')
+        errmsg = exe_err.format(name='dummy_path', kw='sos_triangle')
+        with pytest.raises(OSError, match=errmsg):
+            create_vmd_surface(sos_triangle='dummy_path')
+        errmsg = 'sph_process failed'
+        with pytest.raises(OSError, match=errmsg):
+            create_vmd_surface()
+        with tmpdir.as_cwd():
+            h = hole2.HoleAnalysis(universe)
+            h.run()
+            h.create_vmd_surface()
```

Comment:
Why are these tests necessary? Aren't they already covered in `TestOSError` above?